plugins/mcreator-localization/lang/texts.properties

```diff
@@ -14,7 +14,7 @@ dialog.workspace_selector.vcs_info=<html>Please enter the URL of the repository/
   the Git account you will enter here. Note that Git account is not your MCreator account.<br>
 dialog.workspace_selector.no_workspaces_loaded=<html><center>Failed to load recent workspaces<br>shortcuts. Re-open them to add<br>them to the list again.
 dialog.workspace_selector.no_workspaces=No recent workspaces
-dialog.workspace_selector.version=\  Version {0}
+dialog.workspace_selector.version=\  Version {0}https://github.com/MCreator/MCreator/pull/1547/conflicts
```

Comment:
Why is this here?

Reply:
Surely a CTRL + V I accidentally pressed. xD